src/Compilers/CSharp/Portable/CommandLine/CSharpCompiler.cs

```diff
@@ -267,6 +267,28 @@ public override void PrintLogo(TextWriter consoleOutput)
             consoleOutput.WriteLine();
         }
 
+        public override void PrintLangVersions(TextWriter consoleOutput)
+        {
+            consoleOutput.WriteLine(ErrorFacts.GetMessage(MessageID.IDS_LangVersions, Culture));
+            var defaultVersion = LanguageVersion.Default.MapSpecifiedToEffectiveVersion();
+            var latestVersion = LanguageVersion.Latest.MapSpecifiedToEffectiveVersion();
+            foreach (LanguageVersion v in Enum.GetValues(typeof(LanguageVersion)))
+            {
+                if (v == defaultVersion)
+                {
+                    consoleOutput.WriteLine($"{v.ToDisplayString()} (default)");
+                }
+                else if (v == latestVersion)
+                {
+                    consoleOutput.WriteLine($"{v.ToDisplayString()} (latest)");
+                }
+                else if (v != LanguageVersion.Default && v != LanguageVersion.Latest)
```

Comment:
Also, we probably should think how to make it clear that ```default``` and ```latest``` aren't statically linked to the versions they are mapped in this version of the compiler.